added a list of initial nuclei configurations and modified modules

kosukesato opened the pull request:
Modified the way to start propagation of nuclei and electronic states.
I have already prepared "System" objects for each Initial Nuclei Configurations(INC).
And, I initialize Nuclear and electronic and Thermostat objects for every INC.
Initial datas from GAMESS output are kept in runMD function.(index 0 is added to all data name.)

output data has also INC index (say, ene0.dat, md1.xyz, etc...)

kosukesato commented:
In run.py, some parameters for Surface Hopping methods are defined, but still not used in the program.
Later, I'll add modules using these.

alexvakimov commented:
Kosuke, please proceed with the TSH implementation. I'll review and merge
the code once it is included and tested. You can actually start with just
one cofiguration (one initial nuclear state) - to test the implementation.
Then, extend to several initial conditions.

On Sat, Mar 12, 2016 at 11:23 PM, kosukesato <notifications@github.com>
wrote:

> In run.py, some parameters for Surface Hopping methods are defined, but
> still not used in the program.
> Later, I'll add modules using these.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alexvakimov/libra-gamess_interface/pull/11#issuecomment-195868279>
> .
>



-- 

Dr. Alexey V. Akimov

Assistant Professor
Department of Chemistry
University at Buffalo, SUNY
716 Natural Sciences Complex
Buffalo, NY 14260-3000

email: alexeyak@buffalo.edu <alexvakimov@gmail.com>
web:  http://www.acsu.buffalo.edu/~alexeyak/

kosukesato commented:
Added a function which calculates SH populations.
The type of SH can be selected by "SH_type" parameter.
CPA with Boltzmann factor or with velocity scaling can be chosen by the options: "use_boltz_factor" and do_rescaling. (See run.py script)
During the calculation with velocity scaling type, much more memories will be allocated than with Boltzmann factor type.
Some flags to check the SH calculation,say g matrices and SH state etc... , was introduced in the run.py script.

In the present, averaging SE or SH populations over initial nuclei configurations are not implemented yet. 
Seems the code where SH calculation with velocity scaling is too long; so this should be fixed for convenience. 

Modified some documentation on some modules.
